docs/source/troubleshooting.rst

```diff
@@ -31,11 +31,34 @@ Please see :ref:`support`.
 How do I figure out what happened with my failed jobs?
 ------------------------------------------------------
 
-**This applies to AWS only**.
 
-If you find any jobs failed as a result of running ``elastic-blast status``,
+If the ``elastic-blast status`` command reports a jobs as having failed,
 please re-run the command with the additional ``--verbose`` option, which
-will print more information about the jobs that failed, e.g.:
+will print more information about the jobs that failed.  You can ignore the 
+message about reporting the failure unless you find the message with ``--verbose``
+unhelpful.
+
+In the example below, there was a problem with a BLAST option.  The error message from the
+blastn executable returned with the ``--verbose`` option should be sufficient for a user to 
+correct the issue.
+
+.. code-block:: bash
+
+    elastic-blast status --cfg bad_options.ini --verbose
+    Your ElasticBLAST search failed, please help us improve ElasticBLAST by reporting this failure as described in https://blast.ncbi.nlm.nih.gov/doc/elastic-blast/support.html
+    USAGE
+    blastn [-h] [-help] [-import_search_strategy filename]
+     [-export_search_strategy filename] [-task task_name] [-db database_name]
+     [-dbsize num_letters] [-gilist filename] [-seqidlist filename]
+     [-negative_gilist filename] [-negative_seqidlist filename]
+     [-taxids taxids] [-negative_taxids taxids] [-taxidlist filename]
+     [-negative_taxidlist filename] [-entrez_query entrez_query]
+    [...]
```

Comment:
Maybe it would be good to show the full error message or at least the bottom part where the incorrect blast option is shown.

Reply:
I had thought about that, but I'm not sure it will help people that much compared to making the description longer and more difficult to understand.  My error was that I used -expect, but this should have been -evalue.  I'd have to explain that etc.  I'm expecting most users will be able to understand the BLAST+ message.  I'm also thinking that the majority of the EB users will also be BLAST+ users and know how this works.  Anyway, to have written down BLAST options, they'll need to have looked at BLAST+ docs or run BLAST+ before.